mods/d2k/rules/structures.yaml

```diff
@@ -688,6 +693,19 @@ hightech:
 		RaceImages:
 			atreides: hightech.atreides
 			ordos: hightech.ordos
+	ProvidesCustomPrerequisite@UPGRADE:
+		Prerequisite: hightech.atreides
+		Race: atreides
+	AirstrikePower:
+		Icon: ornistrike
+		Description: Air Strike
+		Prerequisites: ~techlevel.superweapons, ~upgrade.hightech
```

Comment:
`upgrade.hightech` already requires `techlevel.superweapons`, so having it as a prerequisite here is probably redundant.